# map_server_params.yaml
map_server:
  ros__parameters:
    yaml_filename: "/home/susheel/ros2_tk/src/task_bot/maps/custommap.yaml"